.github/workflows/deploy.yml

```diff
@@ -65,14 +73,12 @@ jobs:
           load: false
           target: dallinger
           tags: |
-            ghcr.io/dallinger/dallinger:${{ steps.vars.outputs.version }}
-            ghcr.io/dallinger/dallinger:latest
+            ghcr.io/dallinger/dallinger:python3.13
       - name: Build ghcr.io/dallinger/dallinger-bot docker image
         uses: docker/build-push-action@v5
         with:
           push: true
           load: false
           target: dallinger-bot
           tags: |
-            ghcr.io/dallinger/dallinger-bot:${{ steps.vars.outputs.version }}
-            ghcr.io/dallinger/dallinger:latest
+            ghcr.io/dallinger/dallinger-bot:python3.13
```

Comment:
This is only meant to create a Docker image to be used for testing with PsyNet. This needs to be removed once we agree this PR is ready to be merged.